### DISCUSSION TITLE
Error: Failed to delete project: 401

### DISCUSSION THREAD

**Discussion by akyhn** [AntlerVC/firetable#1001] [Support / Q&A] [answered]
👉 **Please describe the steps that you took that led to this bug.**

I just tried to delete the project. I type in the security word in order to delete it and then the error shows up

**Comment by BeeBombshell** [comment, marked as answer]
Hey @akyhn, I'm wondering why this isn't working for you. Perhaps you could email us at [support@rowy.io](mailto:support@rowy.io) with more information so we can assist you further.
